src/middlewares/sellerproduct.middlewares.ts

```diff
@@ -0,0 +1,150 @@
+import { NextFunction, Request, Response } from "express";
```

Comment:
These functionalities should be under product validation or middlewares, there is no need to create another file.